__tests__/user/user.mutation.test.js

```diff
@@ -33,10 +33,18 @@ describe('mutations', () => {
 
         userId = res.data.registerUser._id
         
```

Comment:
Please, add check for the type of `_id` field

Reply:
fixed